noir-projects/noir-contracts/contracts/test/invalid_account_contract/src/main.nr

```diff
@@ -0,0 +1,13 @@
+use aztec::macros::aztec;
+
+/// Used by token contract tests to test interaction with an invalid account.
+#[aztec]
+pub contract InvalidAccount {
```

Comment:
I'm confused - the account is valid, it's the authwits that are invalid

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/13388/commits/803a45d9966f5e74cf9b52b145e1646005c886b6 Thanks